src/deluge/model/global_effectable/global_effectable_for_clip.h

```diff
@@ -59,6 +59,7 @@ class GlobalEffectableForClip : public GlobalEffectable {
 	                  StereoSample* outputBuffer, int32_t numSamples, int32_t* reverbBuffer, int32_t reverbAmountAdjust,
 	                  int32_t sideChainHitPending, bool shouldLimitDelayFeedback, bool isClipActive,
 	                  OutputType outputType, SampleRecorder* recorder);
+	void stopDelay();
```

Comment:
You don't need to override this. It's the same as the base class (GlobalEffectable)